sklearn/preprocessing/tests/test_function_transformer.py

```diff
@@ -30,20 +30,11 @@ def test_delegate_to_func():
     )
 
     # The function should only have received X.
-    assert_equal(
-        args_store,
-        [X],
-        'Incorrect positional arguments passed to func: {args}'.format(
-            args=args_store,
-        ),
-    )
-    assert_equal(
-        kwargs_store,
-        {},
-        'Unexpected keyword arguments passed to func: {args}'.format(
-            args=kwargs_store,
-        ),
-    )
+    assert args_store == [X], ('Incorrect positional arguments passed to '
+                               'func: {args}'.format(args=args_store))
+
+    assert not kwargs_store, ('Unexpected keyword arguments passed to '
```

Comment:
strictly speaking, `not kwargs_store` is `True` if `kwargs_store == False`, or `kwargs is None`. I think it's safer if we check it to be exactly what we want.

Reply:
I'm not sure I follow: `kwargs_store` is a dict? We just check that it's empty

You mean you want  `assert kwargs_store == {}`?

Reply:
Yeah I prefer the `assert kwargs_store == {}`, but you're right that here it's a dict. So never mind.

This LGTM then.